tests/jballerina-unit-test/src/test/resources/test-src/types/intersection/error_intersection_type_negative.bal

```diff
@@ -56,3 +56,12 @@ function testRecordAndMapIntersection() {
     var err = error IntersectionErrorFour("message", x = "x", z = 10);
     DistinctErrorIntersection err2 = error IntersectionErrorFour("message", x = "x");
 }
+
+type X error<record { int i?; }> & error<record { int i; }>;
+type Y readonly & record { int i; };
```

Comment:
Shall we move the record scenarios to non-error intersection test? Shall we also add a positive test separately?

Reply:
Positive test as in? For record intersections?